dashboard/src/components/homepage/HomePageSlice.js

```diff
@@ -7,7 +7,7 @@ const homePageSlice = createSlice({
       {
         title: "Training Datasets",
         icon: "storage",
-        path: "/training-datasets",
+        path: "/training-sets",
```

Comment:
This is very similar logic to the map code above, why not change it to all use the same logic so you don't have to change the path in multiple different places.